README.md

```diff
@@ -142,6 +143,52 @@ Browserslist will take queries from tool option,
   than Microsoft Edge. Chinese QQ Browsers has more market share than Firefox
   and desktop Safari combined.
 
+## Why you need to call update-db script regularly?
+
+This is a core problem with npm architecture. It happens with lodash (a lot of
+modern webapps are bloated by loadash duplicates.)
+
+Just google [npm deduplication](https://www.google.com/search?q=npm+deduplication).
```

Comment:
Copy-pasting will not work, since my comment was written for person-to-person context and not in docs style.

Creating good docs is hard work :(. Sorry for bothering you will extra requirements.

Reply:
No way! I'm happy to do this myself. I misunderstood you earlier and thought that you wanted to use what you wrote up and I took it and edited it to fix some grammar issues. Let me revise it a bit more.